src/public/modules/forms/helpers/csv-response-classes/ArrayAnswerResponse.class.ts

```diff
@@ -0,0 +1,20 @@
+import { ArrayResponse } from '../../../../../types/response'
+
+import { Response } from './Response.class'
+
+export class ArrayAnswerResponse extends Response {
+  _data: ArrayResponse
```

Comment:
can probably mark this as private with javascript's private modifier [`#`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) and remove the underscore hehe

Reply:
Not possible due to ECMA version :(
```
ERROR in /home/travis/build/opengovsg/FormSG/src/public/modules/forms/helpers/csv-response-classes/SingleAnswerResponse.class.ts(6,3)
          TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.
```

Reply:
ehhhhh we are targeting ES6 (i.e. ECMAScript 2015) in our TSConfig....

Reply:
use Typescript's [`private`](https://www.typescriptlang.org/docs/handbook/classes.html#public-private-and-protected-modifiers) modifier then...